app/services/trainees/set_academic_cycles.rb

```diff
@@ -62,6 +62,9 @@ def actual_course_duration
       course_duration_amount.public_send(course_duration_unit)
     end
 
+    # The fields study_length and study_length_unit were removed from HESA from
```

Comment:
Should we add this comment to `Trainees::CalculateIttEndDate` since that's where the code lives now?